pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -76,7 +76,7 @@ sub generate_advice {
 
         # These checks will only run on v88 and highger.
         if ( Cpanel::Version::compare( $cpanel_version, '>=', $IMUNIFYAV_MINIMUM_CPWHM_VERSION )
-            && ( !$self->{i360}{installed} && !$self->{i360}{licensed} ) ) {
+            && !$self->{i360}{installed} ) {
```

Comment:
we should probably check for `$self->{i360}` also

Reply:
done